frontend/app/(withSidebar)/home/page.tsx

```
@@ -0,0 +1,623 @@
+    processed_documents: number;
+    ai_insights: number;
+  };
+  recent_active_projects: Array<{
+    id: string;
+    title: string;
+    updated_at: string;
+  }>;
+}
+
+interface Activity {
+  type: string;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Activity'. Consider to delete it or rename it.

'Activity' is defined here:


(lint/suspicious/noRedeclare)